### PR TITLE
Update security-severity scores

### PR DESCRIPTION
**Pull request by calumgrant**
Run using latest changes to the CWE tooling (https://github.com/github/cwe-scores).

The change is that the score is based on the CVE "Base Score" instead of its "Impact score".

The result of this is that the scores map better to the UI concept of (Low/Medium/High/Critical).

### CODE REVIEW

**Comment by calumgrant** [comment]
The distribution of the scores are as follows:

| Language | Low | Medium | High | Critical |
|----|---|---|---|---|
| All | 9 | 144 | 332 | 138 |
| C# | 1 | 22 | 64 | 24 |
| C++ | 2 | 13 | 83 | 33 |
| Golang | 0 | 13 | 22 | 10 |
| Java | 0 | 60 | 95 | 46 |
| JavaScript | 6 | 37 | 56 | 26 |
| Python | 0 | 12 | 34 | 9 |

(Golang is in a separate repo and is not included in this PR).

File: cpp/ql/src/Likely Bugs/Memory Management/SuspiciousCallToStrncat.ql [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * @description Calling 'strncat' with an incorrect size argument may result in a buffer overflow.
  * @kind problem
  * @problem.severity warning
- * @security-severity 10.0
+ * @security-severity 9.3
```
Comment (rdmarsh2) [inline cpp/ql/src/Likely Bugs/Memory Management/SuspiciousCallToStrncat.ql:6]: I'm surprised to see some of these drop - is that explained by the formula that converts base scores into impact scores?

**Review by aschackmull — APPROVED** [review]
From a quick look through, this looks like an improvement.

**Review by MathiasVP — APPROVED** [review]
These scores reflect my expectations much better 👍! Like @rdmarsh2, I'm curious about why some of the scores are slightly lowered by this change, though.
(It's not a big deal since none of the drops cause a query to change to a lower category.)

**Review by RasmusWL — APPROVED** [review]
:+1: For python

File: cpp/ql/src/Likely Bugs/Memory Management/SuspiciousCallToStrncat.ql (hunk 1 above)
Comment (calumgrant) [reply cpp/ql/src/Likely Bugs/Memory Management/SuspiciousCallToStrncat.ql:6]: The CVE base score tends to be higher than the impact score, but not always. Impact = real-world exploitability, which is often higher in C++ due to the ability to run shell code. Looking at the list, the 75th percentile CVE has base score of 9.3 and impact score of 10.0. You can use the `explain.py` tool to dig into strange results.

I'm not concerned or surprised by this result as the category hasn't changed and feels right for the query.

**Comment by sj** [comment]
Thanks @calumgrant! Is it possible to easily generate an overview like this for us to do an extra sanity check:

| security-severity \ severity | recommendation | warning | error |
| ------------- | ------------- | --- | --- |
| **low**  | 1  | 2 | 3 |
| **medium**  | 4 | 5 | 6 |
| **high**  | 7  | 8 | 9 |
| **critical**  | 10 | 11 | 12 |

This will allow us to check how many things that we previously gave a high severity (i.e., `error`) actually end up having a low security-severity.

(cc @AlonaHlobina)

**Pull request merged**